docs/version0.44.md

```diff
@@ -76,6 +77,14 @@ The `jcmd` tool also now supports specifying `0` as a VMID to target all VMs.
 
 For more information, see [Java diagnostic command (`jcmd`) tool](tool_jcmd.md).
 
+### Flag names that refer to the fields of `J9BuildFlags` changed in the DDR code
+
+The Direct Dump Reader (DDR) code enables the VM to read the system dump data by using the OpenJ9 Diagnostic Tool Framework for Java (DTFJ) API or the [`jdmpview`](tool_jdmpview.md) tool. The DDR code provides many built-in commands. You can extend the DDR code, adding your own commands, by writing plug-ins. These user plug-ins might contain code that has `J9BuildFlags` reference to read the build flags in the system dump data.
+
+When the `J9BuildFlags` is generated for each build, the raw name of the build flags might be used in the plug-in code rather than the `cname` that appears in `j9cfg.h` (derived from `j9cfg.h.ft1`). Therefore, mapping of flag names in the plug-in code with the names that are specified in `j9cfg.h` was inconsistent. Introducing new flags meant either extra complexity or that some flag tests might not apply to all system dumps (because the flag name in `J9BuildFlags` was different at different times).
```

Comment:
Code that "might use" a raw name won't compile.
The root source file is `j9cfg.h.ftl` (the last letter is a lowercase `L`, not a `1`).

Reply:
Regarding your first comment "Code that "might use" a raw name won't compile.", where exactly am I supposed to make the change? Is it the last sentence?

Also, is there any other term that we can use for "raw name"? What exactly is meant by "raw name of the build flags"?

Thanks!